script/zmb

```diff
@@ -270,6 +270,34 @@ sub cmd_test_progress {
 }
 
 
+=head2 get_test_params
+
+ zmb [GLOBAL OPTIONS] get_test_params [OPTIONS]
+
+ Options:
+   --test-id TEST_ID
```

Comment:
It looks like other methods use `--testid` (see `test_progress` and `get_test_results`). Could you change this one to be consistent with the other ? (Or the other way around).

Reply:
Outside this change I find two `--testid` and one `--test-id`. I agree that making them consistent would be good. I might also affect `zmtest` for which `zmb` is a backend.

Reply:
I agree that this sucks, but I actually made it so in order to be consistent with the [RPCAPI](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/API.md#api-method-get_test_params).

Reply:
I think we can start by making it consistent in zmb because it is more likely that you do things manually there. Or support both.

Reply:
Could you then update all the other one to `--test-id` in this PR, so they all evolve together ?

Reply:
I just checked and it's inconsistent all over the place. Maybe we should start migrating to `testid` for test ids across the board. I'll deprecate all `test-id`, `test_id` and `id` that I can find.

Reply:
I created #790 to better facilitate such a migration. The migration itself is out of scope for this PR.